addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AwsAccountUpdateContent.js

```diff
@@ -220,19 +231,52 @@ class AwsAccountUpdateContent extends React.Component {
             CloudFormation console where you can review the stack information and provision it.
             <div className="mb0 flex mt2">
               <div className="flex-auto">
-                <Button fluid as="a" target="_blank" href={createStackUrl} rel="noopener noreferrer">
+                {willShowWarning && (
+                  <Message warning>
+                    <Message.Header>Caution!</Message.Header>
+                    <p>
+                      Be advised that deploying a new CFN stack may cause any workspaces associated with this account in
+                      SWB to become unusable. To proceed, please acknowledge the warning below.
+                    </p>
+                  </Message>
+                )}
+                {willShowWarning && (
+                  <Checkbox
+                    label="I am aware that re-onboarding this account may render workspaces associated with this account to become unusable."
+                    onClick={this.handleClickAcknowledgement}
+                  />
+                )}
+              </div>
+            </div>
+            <div className="mb0 flex mt2">
+              <div className="flex-auto">
+                <Button
+                  fluid
+                  as="a"
+                  target="_blank"
+                  href={createStackUrl}
+                  disabled={!warningAcknowledged}
```

Comment:
We can remove the line at 54. And this can be
`disabled={willShowWarning && !warningAcknowledged}`